### ISSUE TITLE
Incorrect Model passed to View when Hijacking Route in Umbraco Headless

### ISSUE THREAD

**Issue by emmanueltissera** [umbraco/UmbracoDocs#1466]
## What type of issue is it? (Choose one - delete the others)
Wrong documentation

## What article/section is this about? 
https://github.com/umbraco/UmbracoDocs/blob/master/Umbraco-Cloud/Headless/Headless-Net-Client/website-managed.md
`Hijacking routes` section

## Describe the issue:
In the code sample  
`var content = UmbracoContext.GetContent();` returns a `ContentItem` whereas the view expects the model to be of type `Page`. How does this conversion happen?
Previously, I could do `var model = HeadlessService.MapTo<Page>(content);`

The code sample itself is incorrect whereas it retrieves `content` but passes a `model` to the view.

**Comment by sitereactor** [comment]
Thanks for submitting this issue. We rebuilt the .NET Client library to avoid the use of reflection and better support for using Xamarin with both Android and iOS. During this rebuild we leftout the `MapTo<T>` method from the HeadlessService as per your report. I have added in an equivalent method, so you should be able to use as before in a Web context.

I updated the documentation here: https://github.com/umbraco/UmbracoDocs/blob/master/Umbraco-Cloud/Headless/Headless-Net-Client/website-managed.md
And a new build of the Headless .NET Client is available via myget with version `0.9.7-CI-20190130-01`

```
PM> Install-Package UmbracoCms.Headless.Client.Web -Version 0.9.7-CI-20190130-01 -Source https://www.myget.org/F/uaas/api/v3/index.json
```

**Comment by emmanueltissera** [comment]
Hi @sitereactor ,

All good with this method now. Working as expected.

Cheers,
Emmanuel
